bp_be/src/include/bp_be_dcache/bp_be_dcache_lce_pkg.vh

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
Headers should be a little more descriptive. Maybe we should converge on some format for these.